content/governance.md

```diff
@@ -145,6 +147,34 @@ A vote on a single proposal is considered successful if at least two thirds of t
 
 If there are multiple alternatives, members may vote for one or more alternatives, or vote “no” to object to all alternatives. A vote on multiple alternatives is considered decided in favor of one alternative if it has received the most votes in favor, and a vote from at least two thirds of those eligible to vote. Should no alternative reach this quorum, another vote on a reduced number of options may be called separately.
 
+## On- / Offboarding
+
+Changes in the On- /Offboarding section can be taken by [lazy consensus](#consensus) unless challenged.  If no consensus can be reached, the matter may be resolved by [majority vote](#majority-vote).
+
+### Onboarding
+
+The new member is
+* added to the list of [team members](#team-members). Ideally by sending a PR of their own, at least approving said PR.
+* announced on the [developers mailing list][devs] by an existing team member. Ideally, the new member replies in this thread, acknowleding team membership.
```

Comment:
acknowledging

Reply:
I RAN :set spell ON THE INITIAL COMMIT, BUT NOT THIS ONE :(

Thanks

Reply:
(New computer, vimrc not copied yet)